default.nix

```diff
@@ -1,66 +1,121 @@
-{ frontend ? false }:
+{ ... } @ args:
+
 let
-  obelisk-src = (import <nixpkgs> {}).fetchFromGitHub {
-    owner = "obsidiansystems";
-    repo = "obelisk";
-    rev = "6c38599615eddba1b9f8dfb845f7404f53ed8053";
-    sha256 = "0pqhppn2cb69v7r6wbg2zrx5ylxbgq7bl7qilarfbmyd9gcph9h4";
-  };
-  reflex-platform = (import obelisk-src {}).reflex-platform;
 
-  gargoyle-src = reflex-platform.nixpkgs.fetchFromGitHub {
-    owner = "obsidiansystems";
-    repo = "gargoyle";
-    rev = "2c19c569325ad76694526e9b688ccdbf148df980";
-    sha256 = "0257p0qd8xx900ngghkjbmjnvn7pjv05g0jm5kkrm4p6alrlhfyl";
-  };
-  groundhog-src = reflex-platform.nixpkgs.fetchFromGitHub {
-    owner = "obsidiansystems";
-    repo = "groundhog";
-    rev = "c2f18be45e3233f6268c8468eb0732dd6b2e8009";
-    sha256 = "1r9i78bsnm6idbvp87gjklnr10g7c83nsbnrffkyrn1wmd7zzqdn";
+  obelisk = import ./.obelisk/impl args;
+  reflex-platform = obelisk.reflex-platform;
+  pkgs = obelisk.nixpkgs;
+  inherit (pkgs) lib;
+  haskellLib = pkgs.haskell.lib;
+
+  # Some dependency thunks needed
+  repos = {
+
+    # Not sure why this is needed?
```

Comment:
Let's sort that out and get this upstreamed to reflex-platform/gargoyle

Reply:
It looks like gargoyle can be removed but groundhog is still needed.